Sming/rboot/eagle.rom.addr.v6.ld

```diff
@@ -0,0 +1,347 @@
+PROVIDE ( Cache_Read_Disable = 0x400047f0 );
```

Comment:
I was going to say don't use git to apply the patch just use the patch command, but then you're on windows so you may not have it. The diff is against the feature/rBoot branch in my fork of sming, but if you can't get it going you could just manually apply the changes. There are only about 6 lines to change and two files to delete.